pkg/api/org_users.go

```diff
@@ -157,6 +157,55 @@ func (hs *HTTPServer) getOrgUsersHelper(query *models.GetOrgUsersQuery, signedIn
 	return filteredUsers, nil
 }
 
+// SearchOrgUsersWithPaging is an HTTP handler to search for org users with paging.
+// GET /api/org/users/search
+func (hs *HTTPServer) SearchOrgUsersWithPaging(c *models.ReqContext) response.Response {
+	perPage := c.QueryInt("perpage")
+	if perPage <= 0 {
+		perPage = 1000
+	}
+	page := c.QueryInt("page")
+
+	if page < 1 {
+		page = 1
+	}
+
+	result, err := hs.searchOrgUsersHelper(&models.SearchOrgUsersQuery{
+		OrgId: c.OrgId,
```

Comment:
Looks good, just instead of recreate searchOrgUsersHelper, should be better to reuse getOrgUsersHelper